src/extra/gfx/main_window.cpp

```diff
@@ -126,6 +138,11 @@ struct MainWindow : public Base {
     WindowCreationOptions windowOptions = {};
     windowOptions.width = _pwidth;
     windowOptions.height = _pheight;
+    if (_pwidth == 0 || _pheight == 0) {
+      windowOptions.fullscreen = true;
+    } else {
+      windowOptions.fullscreen = false;
+    }

```

Comment:
One issue with this logic is not being able to display in fullscreen at a lower resolution than the screen. Sometimes for performance reason, you want to be fullscreen at `1920*1200` while your screen native resolution is `2560*1440`.

Reply:
In modern rendering you don't do that anymore actually @Kryptos-FR , just use render targets at lower resolution and render all to texture (which you always do due to post effects) 😉